ConfigReader/ApplicationConfig.properties

```diff
@@ -0,0 +1,11 @@
+# This is file for containing different variables.
+
+URL=https://www.amazon.in/
+category=Amazon Fresh
+
+SenderAddress=ishita.nigam20052017@gmail.com
+SenderPassword=Manno@20
+RecipientsAddress=ishita.nigam@gmail.com,saloninigam17@gmail.com,prabal.rc@gmail.com
+ResultsFilesPath=C:/Users/ishita_nigam/git/AmazonFresh/ResultsData/
```

Comment:
Keep the ResultsFilePath relative or remove any user specific reference.

Reply:
Done.